_extras/guide.md

```diff
@@ -29,6 +29,10 @@ title: "Instructor Notes"
   The advantage of opening a plain `.html` file in the browser is that it
   clearly demonstrates the utility of the `title` element.
 
+## Common Issues
```

Comment:
I like Instructor Notes that are presented episode-by-episode. Can we change the heading to match the title of the episode this note refers to?

Reply:
```suggestion
## Reusing Blocks of Content
```